ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -0,0 +1,935 @@
+             */
+            marker: {
+                radius: 10,
+                lineWidth: 0,
+                symbol: 'circle',
+                fillOpacity: 1,
+                states: {}
+            },
+            link: {
+                /**
+                 * The color of the links between nodes.
+                 *
+                 * @type {Highcharts.ColorString}
+                 * @private
+                 */
+                color: Palette.neutralColor60,
+                /**
+                 * The line width of the links connecting nodes, in pixels.
+                 * @type {number}
+                 *
+                 * @private
+                 */
+                lineWidth: 1,
+                /**
+                 * Radius for the rounded corners of the links between nodes.
+                 * Works for `default` link type.
+                 *
+                 * @private
+                 */
+                radius: 10,
+                cursor: 'default',
+                /**
+                 * Type of the link shape.
+                 *
+                 * @sample   highcharts/series-treegraph/link-types
+                 *           Different link types
+                 *
+                 * @type {'default' | 'curved' | 'straight'}
+                 * @product highcharts
+                 *
+                 */
+                type: 'curved'
+                /**
+                 * Modifier of the shape of the curved link. Works best for
+                 * values between 0 and 1, where 0 is a straight line, and 1 is
+                 * a shape close to the default one.
+                 *
+                 * @default 0.5
+                 * @type {number}
+                 * @since next
+                 * @product highcharts
+                 * @apioption series.treegraph.link.offset
+                 *
+                 */
+            },
+            /**
+             * Options applied to the collapse button.
+             */
+            collapseButton: {
+                onlyOnHover: true,
+                /**
+                 * Whether the button should be visible.
+                 * @default true
+                 */
+                enabled: true,
+                /**
+                 * Offset of the button in the x direction.
+                 * @default 0
+                 */
+                x: 0,
+                /**
+                 * Padding inside the button.
+                 * @default 2
+                 */
+                padding: 2,
+                /**
+                 * Offset of the button in the y direction.
+                 * @default 0
+                 */
+                y: 0,
+                /**
+                 * Height of the button.
+                 * @default 3
+                 */
+                height: 10,
+                /**
+                 * Width of the button.
+                 * @default 3
+                 */
+                width: 10,
```

Comment:
The `@default` tags don't match the value. In fact, I think the `@default` can be skipped, as the docs generator can read it from the code (verify with `npm run jsdoc`).